update crossvalidate caveat and update adapted spark version (#137)
NVIDIA/spark-rapids-examples: examples/Spark-ETL+XGBoost/README.md

    - [Apache Toree Notebook for Scala](../../docs/get-started/xgboost-examples/notebook/toree.md)
    - [Jupyter Notebook with spylon kernel](../../docs/get-started/xgboost-examples/notebook/spylon.md)
    - [Jupyter Notebook for Python](../../docs/get-started/xgboost-examples/notebook/python-notebook.md)
  
For the CrossValidator job, we need to set `spark.task.resource.gpu.amount=1` to submit xgboost training jobs sequentially,

> because if not the customized cross-validator will submit multi xgboost training jobs and spark may schedule the different 

Reviewer: ```suggestion
otherwise the customized cross-validator will submit multiple xgboost training jobs and spark may schedule the different 
```

Author: updated, thanks

> xgboost training tasks into one executor simultaneously which will cause issues.

Reviewer: ```suggestion
xgboost training tasks into one executor simultaneously and trigger 
[issue-131](https://github.com/NVIDIA/spark-rapids-examples/issues/131).
```

Author: updated, thanks

    - [Apache Toree Notebook for Scala](../../docs/get-started/xgboost-examples/notebook/toree.md)
    - [Jupyter Notebook with spylon kernel](../../docs/get-started/xgboost-examples/notebook/spylon.md)
    - [Jupyter Notebook for Python](../../docs/get-started/xgboost-examples/notebook/python-notebook.md)
  

> For the CrossValidator job, we need to set `spark.task.resource.gpu.amount=1` to submit xgboost training jobs sequentially,

Reviewer: ```suggestion
For the CrossValidator job, we need to set `spark.task.resource.gpu.amount=1` to allow only 1 training task running on 1 GPU(executor),
```

Reviewer: I may need to change the statement a little bit after talking with @GaryShen2008  and @wbo4958 for this issue. 
So I changed the wording.

> otherwise the customized cross-validator will submit multiple xgboost training jobs and spark may schedule the different 

Reviewer: ```suggestion
otherwise the customized CrossValidator may schedule more than 1 
```
